plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -119,11 +119,15 @@ func (c *PodSecurityPolicyPlugin) Admit(a admission.Attributes) error {
 		return nil
 	}
 
+	// TODO(liggitt): allow spec mutation during initializing updates?
+	if a.GetOperation() != admission.Create {
```

Comment:
Why not move this check into the `shouldIgnore()`?

Reply:
It's not the same for Admit and Validate. We only skip the mutation on non-Create requests in mutating admission (= Admit).

Reply:
Ok, thanks!

The logic become more and more sophisticated... If you will add a comment here, it would be good addition IMHO.

Reply:
Will do.

Reply:
done